_data/simple-icons.json

```diff
@@ -2708,6 +2708,11 @@
             "hex": "6C47FF",
             "source": "https://clerk.com"
         },
+        {
+            "title": "Clever Cloud",
+            "hex": "171C36",
+            "source": "https://www.clever-cloud.com/app/themes/Starter/assets/img/brand-assets/logo_on_dark.svg"
```

Comment:
```suggestion
            "source": "https://www.clever-cloud.com"
```

Reply:
Thanks, I thought the source URL should point to where the original logo is hosted. I'll commit the logo without the excessive points (sorry I missed them).